src/test/groovy/com/cloudcard/photoDownloader/CSVManifestFileServiceSpec.groovy

```diff
@@ -35,10 +35,11 @@ class CSVManifestFileServiceSpec extends Specification {
             new Photo(id: 1, person: new Person(identifier: "123", email: "test1@example.com"), bytes: null),
             new Photo(id: 2, person: new Person(identifier: "456", email: "test2@example.com"), bytes: null)
         ]
+        csvManifestFileService.photoDirectories = ['/Users/Calvin/downloaded-photos']
```

Comment:
Will this test fail if `/Users/Calvin/downloaded-photos` does not exist?  I prefer to use relative directories within the project's directory structure.